danuluma/andela-project
# #160791487 updated .gitignore file

#### What does this PR do?
It updates .gitignore file to remove unnecessary files

#### Description of Task to be completed?
Updated the gitignore files with files to be omitted during commits. These files should not appear on github

#### How should this be manually tested?
Check out this repo on github. You will not see any of the files listed in .gitignore

#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#160791487

#### Screenshots (if appropriate)
#### Questions:
